accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -13,6 +13,7 @@ After the {gitops-title} Operator is installed, Argo CD automatically creates a
 * Red Hat SSO is installed on the cluster.
 * The {gitops-title} Operator is installed on your {OCP} cluster.
 * Argo CD is installed on the cluster.
+* The `DeploymentConfig` API is available in the cluster. For more information, see "DeploymentConfig [apps.openshift.io/v1]".
```

Comment:
Do we want to link the doc from here?
https://docs.openshift.com/container-platform/latest/rest_api/workloads_apis/deploymentconfig-apps-openshift-io-v1.html

Reply:
Hello @varshab1210 - As per our doc standards, we do not add links in the pre-reqs section and list it separately under the Additional resources section. I have added that link in that section for reference purposes.